tree/dataframe/src/RDFInterfaceUtils.cxx

```diff
@@ -340,16 +340,15 @@ std::vector<std::string> FindUsedColumnNames(std::string_view expression, const
    // To help matching the regex
    const std::string paddedExpr = " " + std::string(expression) + " ";
    static const std::string regexBit("[^a-zA-Z0-9_]");
-   Ssiz_t matchedLen;
 
-   std::vector<std::string> usedBranches;
+   std::set<std::string> usedBranches;
```

Comment:
Would unordered_set be slightly faster?

Reply:
it depends on how many columns are around. Up to a few tens of columns, set wins, otherwise the unordered collection does.